src/js/components/SafeImg.tsx

```diff
@@ -34,7 +34,7 @@ const SafeImg = (props: Props) => {
     // free proxy with a 250 images per 10 min limit? https://images.weserv.nl/docs/
     const originalSrc = props.src;
     if (props.width) {
-      const width = props.width * 2;
+      const width = props.width * 4;
```

Comment:
Images larger than width will still get clipped, presumably. What is the purpose of the `rs:fill` in the imgproxy URL?